### PR TITLE
remove client url + fix mypy error(#115, #101)

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Solve Issues #115, #101

### CODE REVIEW

File: gui/src/utils/index.ts [hunk 1]
```diff
@@ -99,9 +98,6 @@ export const getInitials = (value: string, max = 2): string =>
         .join("")
         .toUpperCase();
 
-export const ENDPOINT =
-    !process.env.NODE_ENV || process.env.NODE_ENV === "development" || process.env.NODE_ENV === "test"
-        ? (process.env.REACT_APP_BACKEND_FLASK_URL as string)
-        : window.taipyUrl;
+export const ENDPOINT = window.location.origin;
```
Comment (FredLL-Avaiga) [inline gui/src/utils/index.ts]: do we really need this ?
Could we not specify all our request without specifying the protocol://host:port part ?
Less is more :-)

**Review by FabienLelaquais — APPROVED** [review]
👌

File: gui/src/utils/index.ts (hunk 1 above)
Comment (dinhlongviolin1) [reply gui/src/utils/index.ts]: if we remove this, I dont know how to make a request :D

Comment (dinhlongviolin1) [reply gui/src/utils/index.ts]: Do you suggest anything else Fred :D

Comment (FredLL-Avaiga) [reply gui/src/utils/index.ts]: If I'm not mistaken any request can be written with "/path/to/resource" and the current protocol, host and port will be used.
even the io socket initialization would take window.location if no URL is specified ...

Comment (FredLL-Avaiga) [reply gui/src/utils/index.ts]: so I'd say just remove ENDPOINT completely

Comment (dinhlongviolin1) [reply gui/src/utils/index.ts]: Thanks Fred. I thought it wouldn't work in axios but it does after I tested it out so all good!

**Review by FredLL-Avaiga — APPROVED** [review]
🥇

**Pull request merged**